cypress/integration/test_table.ts

```diff
@@ -254,20 +255,57 @@ describe("Tests Table", () => {
     });
   });
 
-  describe("Scrolling", () => {
-    it("Fetches and appends additional tests to table as the user scrolls", () => {
+  describe("Changing page number", () => {
+    before(() => {
       cy.visit(TESTS_ROUTE);
-      cy.get(".ant-table-body").scrollTo(0, "101%", { duration: 500 });
-      resultsAreFetchedAndRendered({
-        queryName: "TaskTests",
-        responseName: "taskTests.testResults",
-        requestVariables: {
-          cat: "STATUS",
-          dir: "ASC",
-          statusList: [],
-          testName: "",
-          pageNum: 1,
-        },
+    });
+
+    it("Displays the next page of results and updates URL when right arrow is clicked and next page exists", () => {
+      clickOnPageBtnAndAssertURLandTableResults(
+        dataCyNextPage,
+        secondPageDisplayNames,
+        1,
+        dataCyTableRows
+      );
+    });
+
+    it("Does not update results or URL when right arrow is clicked and next page does not exist", () => {
+      clickOnPageBtnAndAssertURLandTableResults(
+        dataCyNextPage,
+        secondPageDisplayNames,
+        1,
+        dataCyTableRows
+      );
+    });
+
+    it("Displays the previous page of results and updates URL when the left arrow is clicked and previous page exists", () => {
+      clickOnPageBtnAndAssertURLandTableResults(
+        dataCyPrevPage,
+        firstPageDisplayNames,
+        0,
+        dataCyTableRows
+      );
+    });
+
+    it("Does not update results or URL when left arrow is clicked and previous page does not exist", () => {
+      clickOnPageBtnAndAssertURLandTableResults(
+        dataCyPrevPage,
+        firstPageDisplayNames,
+        0,
+        dataCyTableRows
+      );
+    });
+  });
+
+  describe("Changing page size updates URL and renders less than or equal to that many rows ", () => {
+    [20, 10, 50, 100].forEach((pageSize) => {
```

Comment:
Is this test less flaky then the one above? They seem to be doing similar things.